gym_sokoban/envs/boxoban_env.py

```diff
@@ -71,8 +59,21 @@ def reset(self, seed=None, options={}):
             zip_ref = zipfile.ZipFile(path_to_zip_file, 'r')
             zip_ref.extractall(self.cache_path)
             zip_ref.close()
-        
-        self.select_room(seed=seed)
+        self.level_files = [f for f in sorted(listdir(self.train_data_dir)) if isfile(join(self.train_data_dir, f))]
```

Comment:
We should assert that these conform to the naming schema for level files, in case we e.g. write extra files to that directory.

We really shouldn't be doing that, but we have been making directories in it with the astar script and might make similar errors in the future